setup.py

```diff
@@ -8,19 +8,22 @@
     'ftw.testbrowser',
     'plone.app.dexterity',
     'plone.app.testing',
+    'plone.app.contenttypes',
     'unittest2',
-    ]
+]
 
 setup(name='ftw.solr',
       version=version,
       description="Solr integration for Plone using collective.solr",
-      long_description=open("README.rst").read() + "\n" + \
-          open(os.path.join("docs", "HISTORY.txt")).read(),
+      long_description=open("README.rst").read() + "\n" +
```

Comment:
I'm not sure if it is because of the deleted slash but if I do longtest I get:
```
[plone5-1.x]%> ~/src/zest_releaser/bin/longtest
ERROR: Error generating html. Invalid ReST.
/var/folders/fd/k_361k_s61j__v5b1hbsft940000gp/T/tmpbvv6x5pm.rst:108: (ERROR/3) Unexpected indentation.
```

Reply:
The errors are in the readme.rst fixed in #145

Reply:
I locally rebased onto the latest 1.x after the merge of #145 and now `longtest` gives no errors anymore

Reply:
👍